src/ui/modules/VideoHero.tsx

```
@@ -0,0 +1,299 @@
+    (data?.type === 'mux' ? mux.error : null);
+
+  return (
+    <section className="relative w-full h-[80vh] bg-gray-900">
+      {/* SEO-friendly metadata */}
+      <div className="hidden">
+        <h1>{data?.title || 'Video'}</h1>
+      </div>
+
+      {!isPlaying ? (
+        // Thumbnail view
+        <div className="relative w-full h-full cursor-pointer bg-black" onClick={handlePlayClick}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)